modules/auxiliary/gather/browser_info.rb

```diff
@@ -25,8 +25,8 @@ def initialize(info={})
               'Description' => 'A web that collects information about the browser.'
```

Comment:
```suggestion
              'Description' => 'A web server that collects information about the browser.'
```

Reply:
I don't think this was applied.

Reply:
No because I applied the previous on same file and now the diff is "outdated". I'll commit it manually

Reply:
Done in dde207731a8abc19eba8e76e5bfa6a1aeba60569